src/App.tsx

```diff
@@ -3,10 +3,15 @@ import Header from './Header/Header';
 import Main from './Main/Main';
 import Footer from './Footer/Footer';
 import ThemeContext from './Store/ThemeContext';
-import { useState } from 'react';
+
+import useLocalStorage from 'use-local-storage';
 
 const App = () => {
-  const [theme, setTheme] = useState('dark');
+  const defaultDark = window.matchMedia('(prefers-color-scheme: dark)').matches;
```

Comment:
Looks good, but I'll have to test it by switching `prefers-color-scheme` in DevTools after merge. Or was it tested? I'm talking about this emulation selector:
![image](https://github.com/Novacci/about-me/assets/34208222/06cea1d4-fc1f-490a-aaab-7fd8269c0913)

Next time please attach images that can confirm it works properly. For example a screenshot where you change this selector to dark, app becomes dark mode, then you switch it to light, app becomes light.

Reply:
When a user enters the website with chosen theme preferences in his browser settings, the application theme will adjust to these preferences on first website visit